### ISSUE TITLE
[Suggestion]:  score board  to track of user progress

### ISSUE THREAD

**Issue by samdiron** [bragefuglseth/keypunch#65]
**Is your suggestion related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

- [ ] This suggestion is compatible with the [GNOME Human Interface Guidelines](https://developer.gnome.org/hig)

**Comment by krzemyczek** [comment]
Yeah, I started use it for fun, but not being able to track progress... Well, It's still fun, but not as much :)
